paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -55,10 +56,18 @@ def add_subparser(subparsers):
             required=True,
         ).completer = lazy_choices_completer(list_instances)
         status_parser.add_argument(
-            '-c', '--cluster',
-            help='The PaaSTA cluster that has the service you want to %s. Like norcal-prod' % lower,
-            required=True,
-        ).completer = lazy_choices_completer(utils.list_clusters)
+            '-c', '--clusters',
+            choices=list_clusters(),
+            help="A comma-separated list of clusters to view. Defaults to view all clusters.\n"
+            "For example: --clusters norcal-prod,nova-prod"
+        ).completer = lazy_choices_completer(list_clusters)
+        status_parser.add_argument(
+            '--cluster',
+            choices=list_clusters(),
+            help="A single cluster to view.\n"
+            " Deprecated! For example: --cluster norcal-prod. Will supersede any use of --clusters."
```

Comment:
In a more immediate sense the use of paasta restart --cluster mycluster --clusters othercluster,othercluster2 will result in the command only operating on mycluster. I'll change the wording to be less ambiguous